hal/lib/athena/handles/IndexedHandleResource.h

```diff
@@ -18,8 +18,8 @@
 
 namespace hal {
 
-constexpr int32_t IndexedResourceIndexOutOfRange = -1;
-constexpr int32_t IndexedResourceNotAllocated = -2;
+constexpr int32_t HandleIndexOutOfRange = -1;
+constexpr int32_t HandleAlreadyAllocated = -2;
```

Comment:
Oh yeah that's right. The old way used handle returns that were public, but I forgot to do that here. Should I just increment the error codes that are already there and add them to error.h?